app/js/Button.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export default class Button extends React.PureComponent {
```

Comment:
It feels a bit complex to create a new component for adding Glyphicons to buttons? If you think this should stick around, I'd go with an API that focuses on the glyph. If no `glyph` prop is provided, don't render a Glyphicon:

```
<Button glyph="export">Export</Button>
```

react-bootstrap's [Button component](https://react-bootstrap.github.io/components.html#buttons) is worth looking at.